packages/botfuel-dialog/src/views/default-view.en.js

```diff
@@ -15,17 +15,18 @@
  */
 
 const logger = require('logtown')('DefaultView.en');
-const TextView = require('./text-view');
+const BotTextMessage = require('../messages/bot-text-message');
+const View = require('./view');
 
 /**
- * Default text view for english.
- * @extends TextView
+ * Default view for english.
+ * @extends View
  */
-class DefaultView extends TextView {
+class DefaultView extends View {
   /** @inheritDoc */
-  getTexts(userMessage, data) {
+  render(userMessage, data) {
     logger.debug('getTexts', userMessage, data);
```

Comment:
logger.debug('render', userMessage, data);

Reply:
thanks, I fixed it!